tasks.py

```diff
@@ -26,3 +28,42 @@ def _migrate_file(url: str, filename: str):
 
     _migrate_file(sources_url, "db_sources.csv")
     _migrate_file(signal_url, "db_signals.csv")
+
+
+@task
+def lint(
+    c, incremental=True, format=True, revision="origin/dev...", diff=False, check=False
```

Comment:
```suggestion
    c, incremental=True, format=False, revision="origin/dev...", diff=True, check=False
```
can we make it so applying changes is not the default?  id rather not have my carefully handcrafted formatting wiped out because i forgot to include a command line argument.

also, are `format` and `diff` mutually exclusive?  do we ever want both to be `True`?  it seems like both `False` would be pointless?

what does `check` do?

Reply:
In hindsight, yea, good call on the no-auto-format. In personal projects I like format on save, but this totally makes sense for our gradual migration.

- `diff` puts `darker` and `black` into "show me the changes you would make" mode
- `lint-diffs` never formats, just complains (so it's essentially always in `diff` mode)
- `format` controls if `darker` or `black` are run (since they're the formatters)
- so you want `format=True` and `diff=True` if you want both `lint-diffs` and `black` to give you their complaints
- `check` makes `darker` or `black` return a non-zero exit code if any changes would be made (but without `diff` it doesn't return changes)